src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -0,0 +1,247 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    internal class SimpleConsoleFormatter : ConsoleFormatter, IDisposable
+    {
+        [ThreadStatic]
+        private static StringBuilder _logBuilder;
+
+        private const string LoglevelPadding = ": ";
+        private static readonly string _messagePadding = new string(' ', GetLogLevelString(LogLevel.Information).Length + LoglevelPadding.Length);
+        private static readonly string _newLineWithMessagePadding = Environment.NewLine + _messagePadding;
+        private IDisposable _optionsReloadToken;
+
+        public SimpleConsoleFormatter(IOptionsMonitor<SimpleConsoleFormatterOptions> options)
+        : base (ConsoleFormatterNames.Simple)
+        {
+            FormatterOptions = options.CurrentValue;
+            ReloadLoggerOptions(options.CurrentValue);
+            _optionsReloadToken = options.OnChange(ReloadLoggerOptions);
+        }
+
+        private void ReloadLoggerOptions(SimpleConsoleFormatterOptions options)
+        {
+            FormatterOptions = options;
+        }
+
+        public void Dispose()
+        {
+            _optionsReloadToken?.Dispose();
+        }
+
+        internal SimpleConsoleFormatterOptions FormatterOptions { get; set; }
+
+        public override void Write<TState>(in LogEntry<TState> logEntry, IExternalScopeProvider scopeProvider, TextWriter textWriter)
+        {
+            LogLevel logLevel = logEntry.LogLevel;
+            ConsoleColors logLevelColors = GetLogLevelConsoleColors(logLevel);
+            string logLevelString = GetLogLevelString(logLevel);
+
+            string timestamp = null;
+            string timestampFormat = FormatterOptions.TimestampFormat;
+            if (timestampFormat != null)
+            {
+                DateTime dateTime = GetCurrentDateTime();
+                timestamp = dateTime.ToString(timestampFormat);
+            }
+            if (timestamp != null)
+            {
+                textWriter.Write(FormatterOptions.SingleLine ? timestamp + ' ' : timestamp);
+            }
+
+            if (logLevelString != null)
+            {
+                textWriter.WriteColoredMessage(logLevelString, logLevelColors.Background, logLevelColors.Foreground, FormatterOptions.DisableColors);
+            }
+
+            StringBuilder logBuilder = _logBuilder;
```

Comment:
The same comment as the Systemd formatter - just write to the textWriter directly instead of making intermediate strings.